### DISCUSSION TITLE
Substituindo links inativos e adicionando outros links de estudos de Python

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#61] [Trilhas de Ensino]
Vejo que tem alguns tópicos que possuem pouco conteúdo e outros possuem links que estão inativos
No tópico 22 o primeiro link não está funcionando, portanto sugiro trocar pelo:  https://brasilmetalhistoria.net/no%C3%A7%C3%B5es-b%C3%A1sicas-de-gerenciamento-de-mem%C3%B3ria-em-python/
Além disso, adicionar o: https://wiki.python.org.br/FuncionamentoGarbageCollector

**Comment by LuizFernando-TC** [comment]
No tópico 18 sugiro adicionar mais alguns materiais de estudos: https://learn.microsoft.com/pt-br/azure/architecture/data-guide/scenarios/csv-and-json
https://medium.com/@renatojlelis/opera%C3%A7%C3%B5es-em-arquivos-csv-com-python-1aee99273fd5
https://medium.com/@renatojlelis/trabalhando-com-json-no-python-1eb0f97c0c50

**Reply by ArtumosGOC** [reply]
Acho interessante adicionar mais conteúdos de JSON e CSV, também acho importante a adição de conteúdos como o tratamento de XML, YAML e suas diferenças.

**Comment by endersonmenezes** [comment]
Gostei de todos os tópicos que o @LuizFernando-TC comentou, o que você acha @thaifurforo ?!

**Reply by thaifurforo** [reply]
Ótimas sugestões @LuizFernando-TC! Parabéns!
Também concordo com @ArtumosGOC sobre a inclusão de materiais para a manipulação de XML e YAML em Python.
Acho que poderíamos fazer um artigo no Blog da Codaqui sobre os conceitos, diferenças, vantagens e desvantagens entre os formatos de uma forma mais geral, que pudesse ser linkado em todas as trilhas, e dentro das trilhas entrar mais no detalhe de como manipular cada um dentro daquela linguagem. O que acha @endersonmenezes ?

**Reply by endersonmenezes** [reply]
Perfeito! @thaifurforo 

Vamos explorar essas coisas em um artigo escrito a três cabeças? Pensa em alguns desafios sobre esses temas e vai passando pro @LuizFernando-TC e para o @ArtumosGOC então poderemos em 1-2 semanas, construir um artigo para o Blog.

Pensei em algo do tipo para uma nova postagem:
1. Abrindo e Fechando Arquivos com Python (.txt, .json, .csv, .xml, .yaml)
2. Trabalhando com Arquivos XML (Desafio 1)
3. Trabalhando com Arquivos YAML (Desafio 2)

Cada um deles ficaria com um desafio e juntamente com o desafio deveria expor o código utilizado e explicar cada parte.

@LuizFernando-TC e @ArtumosGOC vocês aceitam? A gente pode criar os textos em uma Issue -> e a @thaifurforo ajuda jogar isso no blog.

**Reply by endersonmenezes** [reply]
Foi adicionado um novo tópico falando mais sobre arquivos, está em desenvolvimento, o que acham de complementar ele com esses arquivos em especifico?

**Comment by endersonmenezes** [comment]
Foi iniciada um PR pelo @ArtumosGOC abordando essa parte de arquivos: https://github.com/codaqui/institucional/pull/88
